modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -164,23 +166,23 @@ CREATE TABLE IDN_OPENID_REMEMBER_ME (
             PRIMARY KEY (USER_NAME, TENANT_ID))
 /
 CREATE TABLE IDN_OPENID_USER_RPS (
-            USER_NAME VARCHAR(255) NOT NULL,
-            TENANT_ID INTEGER DEFAULT 0,
-            RP_URL VARCHAR(255) NOT NULL,
-            TRUSTED_ALWAYS VARCHAR(128) DEFAULT 'FALSE',
-            LAST_VISIT DATE NOT NULL,
-            VISIT_COUNT INTEGER DEFAULT 0,
-            DEFAULT_PROFILE_NAME VARCHAR(255) DEFAULT 'DEFAULT',
-            PRIMARY KEY (USER_NAME, TENANT_ID, RP_URL))
+			USER_NAME VARCHAR(255) NOT NULL,
+			TENANT_ID INTEGER DEFAULT 0,
+			RP_URL VARCHAR(255) NOT NULL,
+			TRUSTED_ALWAYS VARCHAR(128) DEFAULT 'FALSE',
+			LAST_VISIT DATE NOT NULL,
+			VISIT_COUNT INTEGER DEFAULT 0,
+			DEFAULT_PROFILE_NAME VARCHAR(255) DEFAULT 'DEFAULT',
+			PRIMARY KEY (USER_NAME, TENANT_ID, RP_URL))
 /
 CREATE TABLE IDN_OPENID_ASSOCIATIONS (
-            HANDLE VARCHAR(255) NOT NULL,
-            ASSOC_TYPE VARCHAR(255) NOT NULL,
-            EXPIRE_IN TIMESTAMP NOT NULL,
-            MAC_KEY VARCHAR(255) NOT NULL,
-            ASSOC_STORE VARCHAR(128) DEFAULT 'SHARED',
-            TENANT_ID INTEGER DEFAULT -1,
-            PRIMARY KEY (HANDLE))
+			HANDLE VARCHAR(255) NOT NULL,
+			ASSOC_TYPE VARCHAR(255) NOT NULL,
+			EXPIRE_IN TIMESTAMP NOT NULL,
+			MAC_KEY VARCHAR(255) NOT NULL,
+			ASSOC_STORE VARCHAR(128) DEFAULT 'SHARED',
+			TENANT_ID INTEGER DEFAULT -1,
+			PRIMARY KEY (HANDLE))
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04